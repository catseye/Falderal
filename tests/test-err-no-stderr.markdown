Falderal Test: err-no-stderr
----------------------------

When running a failing test, if the implementation finished with a failing
exit code, but did not provide anything on `stderr`, expect the error message
to be on `stdout`.

    -> Functionality "Error on stdout" is implemented by shell command
    -> "python3 fail.py %(test-body-text) '' 1"

    -> Tests for functionality "Error on stdout"

    | this is the error message
    ? this is the error message
