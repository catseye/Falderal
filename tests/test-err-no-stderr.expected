--------------------------------
Total test runs: 1, failures: 0
--------------------------------
